react/src/components/Text/GradientText.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from '@emotion/styled';
+import React, { CSSProperties, PropsWithChildren } from 'react';
+
+const GradientBgTxt = styled.div`
+	background: var(--app-gradient);
+	-webkit-background-clip: text;
+	-webkit-text-fill-color: transparent;
+	display: inline-block;
+`;
+
+const HoverGradientParent = styled.div`
+	& > *:nth-of-type(1) {
+		display: inline-block;
+	}
+	& > *:nth-of-type(2) {
+		display: none;
+	}
+
+	&:hover > *:nth-of-type(1) {
+		display: none;
+	}
+	&:hover > *:nth-of-type(2) {
+		display: inline-block;
+	}
+`;
```

Comment:
I feel like we should try to ditch styled components here in moving to common because we don't use it very much at all. If possible. Some of the use cases I've seen in the main ui repo can definitely be done with just tailwind... not sure about this one though. But would be better in my opinion.

Reply:
I think using tailwind instead would be way better DX. If we can change to tailwind easily without breaking then works for me. I can give that a shot.

Reply:
Alright sir I made the switch.